src/devices/LedDisplay/Enums/Alignment.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.LedDisplay.Enums
```

Comment:
should this be `Iot.Device.SegmentDisplay`?

Reply:
The same driver supports and is sold in configurations with LED matrices: https://www.adafruit.com/?q=mini%208x8%20backpack

Reply:
sounds good then, didn't know that!